embedded-webextension-bootstrapped/step1-hybrid-addon/webextension/background.js

```diff
@@ -0,0 +1,16 @@
+"use strict";
+
+browser.storage.local.get("super-important-user-setting")
```

Comment:
@wbamberg there's no reason, I was using the "chrome" namespace everywhere, but using the "browser" on the both sides (in the legacy part and the embedded webextension) looks nicer, but I missed to change a couple of "chrome" API methods call, I'm going to rename them to "browser" and update this PR.